Equipment/EquipmentDb/Misc/rings.xml

```diff
@@ -12,6 +12,30 @@
 		</source>
 	</item>
 
+	<item id="13001" phase="1">
+		<info name="Maiden's Circle" type="RING" slot="RING" unique="no" req_lvl="55" item_lvl="61" quality="RARE" boe="yes" icon="Inv_jewelry_ring_12.png"/>
+		<stats>
+			<stat type="INTELLECT" value="7"/>
+			<stat type="SPIRIT" value="6"/>
+			<stat type="SPELL_DAMAGE" value="18"/>
+		</stats>
+		<source>
+			Drops from multiple high level bosses.
+		</source>
+	</item>
+
+	<item id="12543" phase="1">
+		<info name="Songstone of Ironforge" type="RING" slot="RING" unique="no" req_lvl="55" item_lvl="60" quality="RARE" boe="no" icon="Inv_jewelry_ring_05.png"/>
+		<stats>
```

Comment:
Easy to miss, but this ring and Magni's Will needs a \<mutex> element added to them so equipping either unequips the other if it is equipped. This is to ensure no impossible equipment setups are chosen. See e.g. Mason's Fraternity Ring at the top of this file for an example.